doc/manual/manual.tex

```diff
@@ -1568,7 +1568,41 @@ \section{Installation}
 This is a brief explanation of how to install all the required software and
 \aspect{} itself.
 
-\subsection{Prerequisites}
+\subsection{System prerequisites}
+
+In order to install \aspect{} and its dependencies, you should
+have your system set up to be able to compile and link programs. Additionally,
+\aspect{} needs a number of widely used libraries that are available
+for most operating systems. Therefore, you will need compilers for C, C++ and
+Fortran, the GNU make system, the CMake build system, development versions of
```

Comment:
Without fortran the cmake configuration of trilinos-11.8.1 fails on my system ("The fortran compiler "/usr/bin/mpif90" was not able to compile a simple test program"). Maybe it is related to what MPI compilers are delivered in the MPI package? Anyway, it seems like on Ubuntu this is necessary.